<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vaccipy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bestätigen</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="header_layout">
     <item>
      <widget class="QLabel" name="header_label">
       <property name="text">
        <string>Hier kannst du bestimmte Tage und Uhrzeiten für die Terminbuchung eingrenzen</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="uhr_frame">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="uhr_layout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="uhr_header_label">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Nur Uhrzeiten Zwischen</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="uhrzeiten_layout" stretch="0,0,0,0,0">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>14</number>
        </property>
        <item>
         <widget class="QTimeEdit" name="start_time">
          <property name="currentSection">
           <enum>QDateTimeEdit::HourSection</enum>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
          <property name="time">
           <time>
            <hour>0</hour>
            <minute>1</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="uhr_start_label">
          <property name="text">
           <string>Uhr</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="und_label">
          <property name="text">
           <string>und</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="end_time">
          <property name="time">
           <time>
            <hour>23</hour>
            <minute>59</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="uhr_end_label">
          <property name="text">
           <string>Uhr</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="tage_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>217</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="tage_layout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="wochentage_label">
        <property name="text">
         <string>Nur an folgenden Tagen:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mo_check_box">
        <property name="text">
         <string>Montags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="di_check_box">
        <property name="text">
         <string>Dienstags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mi_check_box">
        <property name="text">
         <string>Mittwochs</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="do_check_box">
        <property name="text">
         <string>Donnerstags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fr_check_box">
        <property name="text">
         <string>Freitags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sa_check_box">
        <property name="text">
         <string>Samstags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="so_check_box">
        <property name="text">
         <string>Sonntags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="termine_frame">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="termine_layout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="termine_anwenden_label">
        <property name="text">
         <string>Bei welchen Terminen sollen die Bedingungen gelten?</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="erster_termin_check_box">
          <property name="text">
           <string>1. Termin</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="zweiter_termin_check_box">
          <property name="text">
           <string>2. Termin</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
